virttest/virt_vm.py

```diff
@@ -468,9 +468,10 @@ def __init__(self, model=None, vendor=None, flags=None, family=None,
         :param smp: set the number of CPUs to 'n' [default=1]
         :param maxcpus: maximum number of total cpus, including
                         offline CPUs for hotplug, etc
-        :param cores: number of CPU cores on one socket
+        :param cores: number of CPU cores on one socket (for PC, it's on one die)
         :param threads: number of threads on one CPU core
         :param sockets: number of discrete sockets in the system
+        :param dies: number of CPU dies on one socket (for PC only)
```

Comment:
Would you please reorder the sequence of parameters to 'sockets, dies, cores, threads'? I guess it makes more sense. Others look good to me.

Reply:
Will update the documentation and smp cmdline to bring them closer to qemu help information.
```
# /usr/libexec/qemu-kvm --help | grep die
-smp [cpus=]n[,maxcpus=cpus][,cores=cores][,threads=threads][,dies=dies][,sockets=sockets]
                cores= number of CPU cores on one socket (for PC, it's on one die)
                dies= number of CPU dies on one socket (for PC only)
```